src/data/featuredContentData.ts

```diff
@@ -0,0 +1,28 @@
+const featuredContentData = {
+  data: [
```

Comment:
You could make `featuredContentData` the array itself so you can call `featuredContentData[randomNumber]` instead of first checking for the `.data` property. Unless if you added this property for another reason?